### PR TITLE
[consensus] replace StorageRead with LibraDBTrait

### PR DESCRIPTION
**Pull request by zekun000**
## Motivation

There's plan to deprecate teh StorageRead in the codebase in favor of
LibraDBTrait to get rid of rpc and async calls.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Y

## Test Plan

Existing tests.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: storage/libradb/src/lib.rs [hunk 1]
```diff
@@ -1023,6 +970,60 @@ impl LibraDBTrait for LibraDB {
             AccountStateProof::new(txn_info_accumulator_proof, txn_info, sparse_merkle_proof),
         ))
     }
+
+    fn get_startup_info(&self) -> Result<Option<StartupInfo>> {
```
Comment (davidiw) [inline storage/libradb/src/lib.rs]: is this just a copy paste?

Comment (zekun000) [reply storage/libradb/src/lib.rs]: yup

**Comment by thefallentree** [comment]
yay, no more network loop! :-D

**Pull request closed without merging**